portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -54,7 +61,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         response.sendRedirect("/comments.html");
     }
     private Comment getComment(HttpServletRequest request) {
-        Comment comment = new Comment(request.getParameter(NAME_PARAMETER), request.getParameter(COMMENT_PARAMETER));
+        long timestamp = System.currentTimeMillis();
```

Comment:
Like I sent to Zoe, I wouldn't use `System.currentTimeMillis();`, instead use the [java.time](https://docs.oracle.com/javase/8/docs/api/java/time/package-summary.html)